_sass/leaderboard.scss

```
@@ -27,7 +27,7 @@
     max-width: 90%;
     padding: 0.9rem 1.4rem;
 
-    border: 2px solid var(--brand-color-secondary);
+    border: 2px solid var(--brand-color-primary);
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)